app/controlplane/pkg/policies/policyprovider.go

```diff
@@ -150,19 +159,59 @@ func (p *PolicyProvider) queryProvider(url *url.URL, digest, orgName, token stri
 		return "", fmt.Errorf("error unmarshalling policy response: %w", err)
 	}
 
-	// extract the policy payload from the query response
-	jsonPolicy, err := json.Marshal(response.Data)
-	if err != nil {
-		return "", fmt.Errorf("error marshalling policy response: %w", err)
-	}
+	// if raw message is provided, just interpret it as a base64 encoded string
+	if response.Raw != nil {
+		if err := unmarshalFromRaw(response.Raw, out); err != nil {
+			return "", fmt.Errorf("error unmarshalling policy response: %w", err)
+		}
+	} else if response.Data != nil {
+		// extract the policy payload from the query response
+		jsonPolicy, err := json.Marshal(response.Data)
+		if err != nil {
+			return "", fmt.Errorf("error marshalling policy response: %w", err)
+		}
 
-	if err := protojson.Unmarshal(jsonPolicy, out); err != nil {
-		return "", fmt.Errorf("error unmarshalling policy response: %w", err)
+		if err := protojson.Unmarshal(jsonPolicy, out); err != nil {
+			return "", fmt.Errorf("error unmarshalling policy response: %w", err)
+		}
 	}
 
 	return response.Digest, nil
 }
 
+func unmarshalFromRaw(raw *RawMessage, out proto.Message) error {
+	switch raw.Format {
+	case "FORMAT_JSON":
```

Comment:
Is the actual format coming in lowercase or uppercase?

Reply:
upcase, that's how protobuf does it. 
```
GET /policies/my-policy
{
  "digest": "sha256:...",
  "raw": {
    "format": "FORMAT_JSON",
    "body": "eyJhcGlWZXJzaW9uIjoid29ya2Zsb3djb250cmFjdC5jaGFpb..."
  }
}
```